### PR TITLE
Prepare 1.1 release

### PR DESCRIPTION
**Pull request by bakura10**
Hi !

This PR removes all features that was deprecated in pre 1 version (as per comment in the code).

Now that 5.5 is not supported, I've raised dependency to 5.6 and switch to variadic where possible.

### CODE REVIEW

**Comment by weierophinney** [comment]
@bakura10 Would you have time to rebase this, by any chance?

**Comment by bakura10** [comment]
Hey ! Normally danizord did it :).

**Comment by weierophinney** [comment]
> Hey ! Normally danizord did it :).

What does that mean, @bakura10 ? /cc @danizord

**Comment by danizord** [comment]
@weierophinney he granted me access to the fork and I've rebased it :)

File: src/Application.php [hunk 1]
```diff
@@ -187,8 +178,9 @@ public function __call($method, $args)
                 ));
         }
 
-        // @TODO: we can use variadic parameters when dependency is raised to PHP 5.6
-        return call_user_func_array([$this, 'route'], $args);
+        $callback = [$this, 'route'];
+
+        return $callback(...$args);
```
Comment (webimpress) [inline src/Application.php]: Can we have here just `$this->route(... $args)`? Isn't it the same?

BTW. About coding standards - should be space after `...` or not?

**Comment by danizord** [comment]
BTW @weierophinney, does SemVer allow to remove deprecated stuff in minor versions? Or we are not following SemVer here?

**Comment by weierophinney** [comment]
In this particular case, we marked functionality created in the beta cycle
as deprecated during the RC cycle, and messaged it was not to be considered
part of the stable API. We even indicated it would be removed for 1.1. As
such, it's an edge case; we kept it in mainly to assist our beta testers in
migrating to the stable release.

On Dec 9, 2016 1:56 PM, "Daniel Gimenes" <notifications@github.com> wrote:

BTW @weierophinney <https://github.com/weierophinney>, does SemVer allow to
remove deprecated stuff in minor versions? Or we are not following SemVer
here?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/zendframework/zend-expressive/pull/376#issuecomment-266107874>,
or mute the thread

**Comment by weierophinney** [comment]
@danizord and/or @bakura10 Could you rebase this against **develop** and re-target the PR, please? There are some substantial changes in the develop release that are the actual target for this.

Thanks!

**Comment by danizord** [comment]
Hey @weierophinney, opened #406 targeting develop :)

**Comment by weierophinney** [comment]
Closing in favor of #406

**Pull request closed without merging**